cpp/src/core/Transpose_Test/Makefile

```diff
@@ -1,4 +1,4 @@
-# Generic makefile for a program compiled with gtest
```

Comment:
This looks great. We will add cmake files and restructure the test directory to automate the testing.